charts/observability-pipelines-worker/templates/podmonitor.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.podMonitor.enabled (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
```

Comment:
Not sure if we want to keep this or assume this will be monitored by a Datadog agent, or self-monitored.

Reply:
Again, not a ton of context on this one, but my assumption is that the only official/supported/recommended monitoring would be via the Datadog agent.